backend/routes/login.js

```diff
@@ -7,18 +7,27 @@ const router = express.Router();
 router.post("/login", async (req, res) => {
   const { email, password } = req.body;
   // res.send({ message: "received in the backend " });
-
+console.log(res)
   // find that user in the database
   const foundUser = await Principle.findOne({ email, password }).exec();
-
+   
+   
   if (foundUser) {
 
     // send the user information to the front end including names , family info .. etc 
     console.log("foundit");
+    console.log(foundUser)
+    res.send(foundUser)
+    
   } else {
     // send error message to the front end 
-    console.log("check the username and password again");
-  }
-});
+    console.log("check the username and password");
+    res.send({message:"false"})
+   
```

Comment:
Could you please add appropriate status code and message. 
This is the case when the user is not in our database, right?